src/compiler/resolutionCache.ts

```diff
@@ -54,9 +56,17 @@ namespace ts {
         refCount: number;
     }
 
+    /*@internal*/
+    export const maxNumberOfFilesToIterateForInvalidation = 256;

```

Comment:
This `export` isn't used it seems?

Reply:
Used in hasReachedResolutionIterationLimit

Reply:
I mean, as an `export` -- you can remove the `export` keyword and this compiles.